Crashlytics/Crashlytics/Models/FIRCLSFileManager.h

```diff
@@ -55,6 +56,7 @@
 - (BOOL)removeItemAtPath:(NSString *)path;
 - (BOOL)removeContentsOfDirectoryAtPath:(NSString *)path;
 - (BOOL)moveItemAtPath:(NSString *)path toDirectory:(NSString *)destDir;
+- (BOOL)didCrashOnPreviousExecution;
```

Comment:
What's the reason for having both `didCrashPreviously` and `didCrashOnPreviousExecution`?

Reply:
Can we call this `crashMarkerFileExists` so we don't confuse it with the public API `didCrashDuringPreviousExecution`

Reply:
Since we call didCrashOnPreviousExecution twice and remove the file marker the first time it's called, it won't be correct the second time.